packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -281,13 +285,19 @@ function serializePayloadParameters(
                     typedParams
                 ),
             } as UpdateContractInput;
+        } case AccountTransactionType.UpdateCredentials: {
+            const updateCredentialsPayload = payload as UpdateCredentialsPayload;
+            return {
+                ...payload,
+                currentNumberOfCredentials: BigInt(0), //TODO: placeholder value, to be populated by wallet calling the account info?
+            } as UpdateCredentialsInput;
```

Comment:
```suggestion
            return payload as UpdateCredentialsInput;
```

The payload will probably contain it, so if you just don't override it, it should be fine.